README.md

```diff
@@ -14,13 +14,24 @@ IDEA theme to IntelliJ
 
     ./gradlew runIde
 
-## How Does It Work?
+## How Does It Work on MacOS?
 
 The plugin polls the macOS Dark Mode configuration by executing the following
 command and interpreting its output.
 
     defaults read -g AppleInterfaceStyle
 
 When the plugin detects that macOS is using Dark Mode, it sets the IDEA theme to
-"Darcula". When the plugin detects that macOS is not using Dark Mode, it sets
-the IDEA theme to "IntelliJ".
+the selected theme. When the plugin detects that macOS is not using Dark Mode, it sets
+the IDEA theme to the corresponding theme.
+
+## How Does It Work on Windows?
+
+The plugin polls the Windows Dark Mode configuration by executing the following
+command and interpreting its output.
+
+    reg query HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Themes\Personalize\AppsUseLightTheme
+
+When the plugin detects that Windows is using Dark Mode, it sets the IDEA theme to
+the selected theme. When the plugin detects that macOS is not using Dark Mode, it sets
```

Comment:
"macOS" should be "Windows"

Reply:
Adjusted README.md to totally fit for Mac and Windows (your requested change and I found that I didn't change the general part)